v0/util/index.js

```diff
@@ -863,6 +863,19 @@ function getFirstAndLastName(traits, defaultLastName = "n/a") {
         : defaultLastName)
   };
 }
+
```

Comment:
We should also check for snake case as-well
Example ` "traits.firstName",
    "traits.firstname",
    "traits.first_name" `

Reply:
@utsabc as per our event spec only camel-case fields are listed. I believe the support for regular fields was retained here and in other places for backward compatibility.
Do you think we need to support other combinations as well?

Reply:
https://github.com/rudderlabs/rudder-transformer/blob/b5dcea4fd840f738a0f25b6a219fc66f44c092e8/v0/util/data/GenericFieldMapping.json#L32
We support snake case as fallback as well in Generic Field mapping, though it's a fallback and this method is generic was thinking it would be better if we handle it there. I

Reply:
Sounds good!

Reply:
@ItsSudip Looks like the TCs were not modified as per the new changes here.